pkg/handler/api.go

```diff
@@ -60,7 +61,7 @@ func RegisterRoutes(engine *echo.Echo) {
 		pgqueue queue.PgQueue
 	)
 	paths := []string{api.FullRootPath(), api.MajorRootPath()}
-	pgqueue, err = queue.NewPgQueue(db.GetUrl())
+	pgqueue, err = queue.NewPgQueue(context.TODO(), db.GetUrl())
```

Comment:
should this be changed to context.Background()? unless you anticipate the source of this context changing

Reply:
wasn't too bad, updated

Reply:
now it should be gone :)